torchtext/experimental/datasets/sequence_tagging.py

```diff
@@ -152,18 +145,21 @@ def CoNLL2000Chunking(*args, **kwargs):
             instance of List
             Default: None
         data_select: a string or tuple for the returned datasets
-            (Default: ('train', 'valid', 'test'))
-            By default, all the three datasets (train, test, valid) are generated. Users
+            (Default: ('train', 'test'))
+            By default, both datasets (train, test) are generated. Users
             could also choose any one or two of them, for example ('train', 'test') or
             just a string 'train'. If 'train' is not in the tuple or string, a vocab
             object should be provided which will be used to process valid and/or test
             data.
 
     Examples:
         >>> from torchtext.datasets.raw import CoNLL2000Chunking
-        >>> train_dataset, valid_dataset, test_dataset = CoNLL2000Chunking()
+        >>> train_dataset, test_dataset = CoNLL2000Chunking()
     """
-    return _setup_datasets(*(("CoNLL2000Chunking", ) + args), **kwargs)
+    if 'data_select' in kwargs:
+        return _setup_datasets(*(("CoNLL2000Chunking", " ") + args), **kwargs)
+    else:
+        return _setup_datasets(*(("CoNLL2000Chunking", " ") + args), **dict(kwargs, data_select=('train', 'test')))
```

Comment:
This immediately needs to be cleaned up in a follow-up PR that also writes out the args/kwargs for cases like this.

Reply:
Update the constructor for those two datasets. Will have a follow up PR to clean up other datasets soon.